src/codec/encoder/PyEncoder.cpp

```diff
@@ -55,6 +55,17 @@ const prtx::EncodePreparator::PreparationFlags ENC_PREP_FLAGS =
                 .cleanupVertexNormals(false)
                 .mergeByMaterial(true); // if false, generation takes ages... 40 sec
                                         // instead of 1.5 sec
+
+IPyCallbacks* getPyCallbacks(prt::Callbacks* cb) {
+#if defined(__APPLE__) && defined(__clang__)
+	// workaround for Apple Clang, dynamic_cast unexpectedly returns zero
+	// probably related to IPyCallbacks beeing an abstract class
+	return static_cast<IPyCallbacks*>(cb);

```

Comment:
Still have a hard time to believe that this is due to IPyCallbacks being abstract. Reserved some time on the weekend to look deeper into what's going on here.